ContactDetailsApi.Tests/V1/UseCase/GetByIdUseCaseTests.cs

```diff
@@ -1,23 +1,78 @@
+using AutoFixture;
+using ContactDetailsApi.V1.Boundary.Request;
+using ContactDetailsApi.V1.Boundary.Response;
+using ContactDetailsApi.V1.Domain;
+using ContactDetailsApi.V1.Factories;
 using ContactDetailsApi.V1.Gateways;
 using ContactDetailsApi.V1.UseCase;
+using FluentAssertions;
 using Moq;
 using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace ContactDetailsApi.Tests.V1.UseCase
 {
+    [TestFixture]
     public class GetByIdUseCaseTests
```

Comment:
Should rename this (and the filename) to reflect the changed use case class name.